### PR TITLE
commit

### PR DESCRIPTION
**Pull request by santiagosan93**
I thought of adding one more line to the about page

### CODE REVIEW

File: app/views/pages/about.html.erb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 <div class="container">
   <h1>This is my about page</h1>
   <p>This is a cool about page made on by Santi</p>
+  <p>This is the next line we're adding</p>
```
Comment (valerieschraauwers) [inline app/views/pages/about.html.erb]: Could you give this p tag a class with `text-center`

**Review by valerieschraauwers — CHANGES_REQUESTED** [review]

File: app/views/pages/about.html.erb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 <div class="container">
```
Comment (valerieschraauwers) [inline app/views/pages/about.html.erb]:
```suggestion
<div class="container mt-4">
```

**Comment by santiagosan93** [comment]
Thanks for the review @valerieschraauwers . Your suggestion have been implemented

**Review by valerieschraauwers — APPROVED** [review]
Looks amazing! 🎉

**Pull request merged**